test/state.test.js

```diff
@@ -91,11 +91,18 @@ test ('.run returns internal state an value', () => {
   eq (run (42) (get), {state: 42, value: 42});
 });
 
-test ('.fantasy-land/ap propagates the initial state to both State instances', () => {
+test ('.fantasy-land/ap propagates the state to both State instances', () => {
   eq (evalState (42) (Z.ap (Z.of (State, x => x), get)), 42);
   eq (evalState (42) (Z.ap (Z.map (constant, get), put (null))), 42);
```

Comment:
How is this test not breaking with the new implementation?

Reply:
Nevermind, the left State is evaluated first. This test probably never made sense.

Reply:
Removed here; 9f29715.